kvmagent/kvmagent/test/unittest_tools/prepare_env.sh

```
@@ -12,6 +12,7 @@ APT_BASED_OS=""
 
 # /root/.zguest/zstack-utility is zstack-utility home of ztest.
 SCRIPTS_HOME="/root/.zguest/zstack-utility/kvmagent/kvmagent/test/unittest_tools"
+MY_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

Shellcheck (0.11.0)

[warning] MY_DIR appears unused. Verify use (or export if used externally).

(SC2034)